charts/argo-templates/templates/export-bq-table.yaml

```diff
@@ -0,0 +1,102 @@
+{{- if (or .Values.exportBQTable.create .Values.diffBQTable.create) }}
+apiVersion: argoproj.io/v1alpha1
+kind: WorkflowTemplate
+metadata:
+  name: {{ default (printf "%s-export" .Values.diffBQTable.name) .Values.exportBQTable.name }}
```

Comment:
Is this to ensure that we don't have a collision with chart names?

Reply:
This prevents the export and diff templates from colliding, but there could be another template with "<diff>-export" as the name. This is more to ensure that all of diff-table's dependencies are rendered, even if they aren't explicitly marked with `create: true` or given a name.